django/chat/eval/glyph_qa.yaml

```diff
@@ -0,0 +1,36 @@
+
+- mode: qa
```

Comment:
Wrote the responses myself based on some preliminary 4o-mini output, and my own understanding of the source material. Obviously you don't want to use pure LLM output as a metric for LLM performance.
Would like a second set of eyes here in particular since I'm no typography expert.